src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -36,6 +36,15 @@
 /* ****************************************************************************
 *
 * postSubscribeContext - 
+*
+* POST /v1/subscribeContext
+* POST /ngsi10/subscribeContext
+*
+* Payload In:  SubscribeContextRequest
+* Payload Out: SubscribeContextResponse
+*
+* URI parameters
+*   - notifyFormat=XXX    (handled by mongoBackend)
```

Comment:
Ah... I have just realize that this is a std-op, so all the URI params for filters are ignored. NTC